### PR TITLE
add library-level docs to help with importing

### PR DESCRIPTION
**Pull request by sethladd**
Helps with https://github.com/flutter/flutter/issues/3119

### CODE REVIEW

File: sdk/lib/collection/collection.dart [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 /**
  * Classes and utilities that supplement the collection support in dart:core.
+ * 
+ * To use this library in your code:
+ *
+ *     import 'dart:collection';
```
Comment (kevmoo) [inline sdk/lib/collection/collection.dart]: Use
```dart
import 'dart:collection';
```

please

Comment (kevmoo) [inline sdk/lib/collection/collection.dart]: ...meaning, triple-tick dart syntax

Comment (floitschG) [inline sdk/lib/collection/collection.dart]: Why?
All of the core libraries use indentation. The only exception seems to be `uri.dart`.

Tbh I don't mind either, but I wouldn't ask patches to the core libraries to use one or the other.

**Comment by sethladd** [comment]
I just followed the previous convention.

**Comment by kevmoo** [comment]
I'm encouraging a slow and steady move towards our current recommendation – but not a biggy

**Comment by sethladd** [comment]
Can we land as is and then change all at once in another PR?

**Comment by sethladd** [comment]
Thanks!

**Pull request merged**